core/src/main/java/bio/terra/pearl/core/model/workflow/AdminTask.java

```diff
@@ -0,0 +1,35 @@
+package bio.terra.pearl.core.model.workflow;
+
+import bio.terra.pearl.core.model.BaseEntity;
+import bio.terra.pearl.core.model.participant.Enrollee;
+import bio.terra.pearl.core.model.participant.ParticipantNote;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+
+import java.time.Instant;
+import java.util.UUID;
+
+
+/**
+ * represents a thing that an adminUser can do, or has already done.  e.g. a participant note might be assigned to
+ * a staff member to follow-up
+ */
+@Getter
+@Setter
+@SuperBuilder
+@NoArgsConstructor
+public class AdminTask extends BaseEntity {
+    private Instant completedAt;
+    @Builder.Default
+    private TaskStatus status = TaskStatus.NEW;
+    private UUID studyEnvironmentId;
+    private UUID enrolleeId;
+    private UUID participantNoteId;
+    private UUID creatingAdminUserId;
+    private UUID assignedAdminUserId;
+    private String description;
```

Comment:
I figured `description` might be useful for tasks that are attached to things that aren't notes (e.g. kit requests).  You're right it's not used yet

Reply:
I brought this up in a separate comment, but I'm wondering if the "Note" field on a task, might be confusing, especially since it seems like there is not a way to edit notes (but I could have missed that)?
You can imagine without note editing, the note field in an associated task becomes a way to update a note, but I assume the "Task" note field is really designed to supplement the tasks, like "Can you complete this by Friday".
Not sure if this is a terminology thing, a design thing, or me not understanding. :-)

Reply:
I'm not sure which "note" field you mean -- tasks can be linked to ParticipantNotes?  But not all tasks are linked to ParticipantNotes, and so there's a need for multiple ways to specify text